plugin.xml

```diff
@@ -34,7 +34,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         <engine name="cordova-android" version=">=6.3.0" />
```

Comment:
You need to also change this:

```xml
<engine name="cordova-android" version=">=10.0.0" />
```

The file plugin `7.0.0` implemented the `WebViewAssetLoader` proxy handler to try and continue to support something like the `cdvfile` process but in a new manner. The `WebViewAssetLoader` is a part of the `AndroidX` library and cordova-android 10.x is when `AndroidX` was introduced.

If this is not updated, there will be build failures because the older platform versions does not support the latest libraries.

Reply:
Done